numpy/core/fromnumeric.py

```diff
@@ -604,15 +604,20 @@ def _transpose_dispatcher(a, axes=None):
 @array_function_dispatch(_transpose_dispatcher)
 def transpose(a, axes=None):
     """
-    Permute the dimensions of an array.
+    Reverse or permute the axes of an array; returns the modified array.
+    For an array a with two axes, transpose(a) gives the matrix transpose
+    of a.
```

Comment:
I think PEP257 needs a blank line before this sentence.

```suggestion
    
    For an array ``a`` with two axes, ``transpose(a)`` gives the matrix transpose
    of a.
```

Reply:
This comment still applies.

Reply:
Pushed fix